lua/zonkan/init.lua

```diff
@@ -1,13 +1,52 @@
 local window = require("zonkan.window")
+local Input = require("zonkan.ui.input")
```

Comment:
In lua global variables start with capital letter. I think you might want to rename it.

Reply:
no i don't want to rename it 🚶🏻

Reply:
LGTM*2